### PR TITLE
Revoke invite

### PR DESCRIPTION
**Pull request by lastminutediorama**

### CODE REVIEW

File: src/interface/src/app/services/invites.service.ts [hunk 1]
```diff
@@ -53,4 +53,15 @@ export class InvitesService {
       }
     );
   }
+
+  deleteInvite(planningAreaId: number, inviteId: number) {
+    return this.http.delete<Invite[]>(
+      BackendConstants.END_POINT.concat(
+        `/invites/invitations/planningarea/${planningAreaId}/${inviteId}`
```
Comment (lastminutediorama) [inline src/interface/src/app/services/invites.service.ts:60]: I'm still open to changing this endpoint format to just accept the inviteId, if we prefer. 
It's a simple change, obviously, I just want to be democratic about it :D

File: src/interface/src/app/home/share-plan-dialog/share-plan-dialog.component.ts [hunk 2]
```diff
@@ -153,8 +153,26 @@ export class SharePlanDialogComponent {
...
+  }
+
   removeAccess(invite: Invite) {
-    this.showSnackbar(`Removed  ${invite.email}`);
+    this.inviteService
+      .deleteInvite(this.data.planningAreaId, invite.id)
+      .subscribe({
+        next: (result) => {
+          this.showSnackbar(`Removed  ${invite.email}`);
+          this.reloadInvites();
```
Comment (lastminutediorama) [inline src/interface/src/app/home/share-plan-dialog/share-plan-dialog.component.ts:168]: I think related functions were just closing the box, but I thought reloading the invites was a little nicer. Open to changing this.

File: src/interface/src/app/services/invites.service.ts (hunk 1 above)
Comment (lastminutediorama) [reply src/interface/src/app/services/invites.service.ts]: Yeah, I agree. I updated these.

**Review by pflopez — APPROVED** [review]
lgtm!

**Pull request merged**